### DISCUSSION TITLE
How much to simplify icons?

### DISCUSSION THREAD

**Discussion by Crazy-Unicooorn** [Arcticons-Team/Arcticons#1684] [Polls]
Hi, I was making some icons, and I had 2 ideas for this one:

![image](https://github.com/Donnnno/Arcticons/assets/30509883/f201c27a-0786-444a-aca1-8292d450d87f)

Which made me ask, how much should I simplify the icons?

I like both icons, but option 1 is simplier and might fit better with Arcticons overall; while option 2 is closer to the original logo.

**Comment by Donnnno** [comment]
I'll leave it up to you, both look good!

Option 1 is a bit more stylized, I like that

**Comment by autinerd** [comment]
I'm more into option 2, because it introduces less ambiguity than option 1.

**Comment by Donnnno** [comment]
Also, a quick sketch from [our chat](https://matrix.to/#/#arcticons-central:matrix.org) 

![1000151778](https://github.com/Donnnno/Arcticons/assets/31142286/45e92870-7985-4ed6-be69-065f75d02b59)

**Comment by Crazy-Unicooorn** [comment]
Nice thanks for the inputs guys!

Didn't know we had a chat, will join soon
